lib/spaceGenerator.tsx

```diff
@@ -1,23 +1,24 @@
+import path from 'path';
+
 import SwaggerParser from "@apidevtools/swagger-parser";
 import { OpenAPI } from "openapi-types";
 import { basename } from "path";
 import Case from "case";
-import JSZip from "jszip";
```

Comment:
Dead code?

Reply:
@Minivera not sure what you mean? Are you asking me to confirm if this was removed because of the duplicated/dead code I had removed? or pointing out that it is dead code?

Reply:
Or sorry you mean the Case import?

Reply:
I think `Case` is still in use in the file:
https://github.com/GitbookIO/openapi-autodoc/pull/1/files#diff-90c686350a11f83dd14c5f0ad0079c547a6e38c3feb672b328bb834d162490a1R22

Sorry if I've misunderstood the whole thing.

Reply:
Sorry, I was asking if this was removed (And the related code) because it was unused or if we had to remove if for this work. Just want to make sure we don't delete used code.

Reply:
Ok yeah it was removed because it was unused by the CLI itself. This code (i.e zip bundling) was duplicated from https://github.com/GitbookIO/openapi-autodoc/blob/main/pages/index.tsx (which is the other tool) but was never used/imported in `/pages/index.tsx` itself. So it was left there and never used.

I removed it because this code was specific to `pages/index.tsx` itself and also removing it reduce drastically the CLI bundle (because of the jszip import.